app/services/idv/steps/capture_mobile_back_image_step.rb

```diff
@@ -5,7 +5,7 @@ def call
         good, data = assure_id.post_back_image(image.read)
         return failure(data) unless good
 
-        failure_data, _data = verify_back_image
+        failure_data, _data = verify_back_image(:mobile_front_image)
```

Comment:
Thoughts on making `reset_step` a named param. That way this could read like this:

```ruby
verify_back_image(reset_step: :mobile_front_image)
```